### PR TITLE
build api documentation with jsduck

### PR DESCRIPTION
**Pull request by xalopp**

### CODE REVIEW

**Comment by xalopp** [comment]
@mmoll, @calbrecht , @andythedandy , @ritters could you review this commit?

File: karma.conf.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ module.exports = function(config) {
     // list of files / patterns to load in the browser
     files: [
         // from sencha CDN
-        'https://extjs.cachefly.net/ext/gpl/5.1.0/build/ext-all-debug.js',
+        // 'https://extjs.cachefly.net/ext/gpl/5.1.0/build/ext-all-debug.js',
```
Comment (elger) [inline karma.conf.js]: Why is ext-all-debug disabled ?

**Comment by mmoll** [comment]
can I see an example, please? :)

**Comment by elger** [comment]
:+1: I am only uncertain about the commented direkt link to ext-all-debug source

**Comment by xalopp** [comment]
@mmoll just pull after this merge to enjoy the API docu  ;-)

**Comment by xalopp** [comment]
@elger I'll delete the commented link 
and switch from ext-all.js to ext-all-debug.js
to avoid confusion

**Comment by mmoll** [comment]
@xalopp I don't see a full-service URL in the README I can click on. :rage4:

**Comment by xalopp** [comment]
@mmoll that's another improvement I'm still working on. 
See https://github.com/xalopp/ExtJsPackages/tree/ghp

Currently it's doing mostly the same as described in
http://www.hoverbear.org/2015/03/07/rust-travis-github-pages/
but I'll also to investigate 
https://github.com/kmcallister/travis-doc-upload/blob/master/README.md
to see what's the best solution.

**Comment by mmoll** [comment]
Glad to hear you're working on getting everything fully serviced. +:100:

**Pull request merged**